.github/workflows/reusable-workflow.yml

```diff
@@ -42,7 +45,7 @@ jobs:
         env:
           # conditionals based on github event
           SYNPRESS_PROFILE: ${{ inputs.is_emerynet_test && 'daily-tests' || 'synpress' }}
-          CYPRESS_AGORIC_NET: ${{ inputs.is_emerynet_test && 'emerynet' || 'local' }}
+          CYPRESS_AGORIC_NET: ${{ inputs.AGORIC_NET }}
```

Comment:
this condition is based on more than the value of inputs.AGORIC_NET it is also dependant on the github.event_name for some workflows such as  the vaults.yml workflow